hc/settings.py

```diff
@@ -146,6 +146,14 @@
 COMPRESS_OFFLINE = True
 
 EMAIL_BACKEND = "djmail.backends.default.EmailBackend"
+DJMAIL_REAL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+# Email
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'hckalii2018@gmail.com'
```

Comment:
@Sebuliba-Adrian @Jak-Sidious  pushing sensitive information such as email usernames and passwords is not good practice. To solve for this, you store the configurations in the env and retrieve in the code.
Kindly rectify this